webhook/certificates/certificates.go

```diff
@@ -68,7 +71,18 @@ func (r *reconciler) reconcileCertificate(ctx context.Context) error {
 		logger.Infof("Certificate secret %q is missing key %q", r.secretName, certresources.CACert)
 	} else {
 		// It has all of the keys, it's good.
-		return nil
+		cert, err := tls.X509KeyPair(secret.Data[certresources.ServerCert], secret.Data[certresources.ServerKey])
+		if err != nil {
+			logger.Infof("error creating pem from certificate and key")
+		}
+		certData, err := x509.ParseCertificate(cert.Certificate[0])
+		if err != nil {
+			logger.Infof("error parsing certificate: ", err.Error)
+		}
+		oneWeek := 168 * time.Hour
```

Comment:
I believe you can make this a const

Reply:
Will do. I'm open to suggestions to a shorter period of time as well, I just went with 1 week instead of 48 hours.